### ISSUE TITLE
 Bug in Federated Learning on MNIST using a CNN

### ISSUE THREAD

**Issue by sreyafrancis** [OpenMined/PySyft#1978]
Hi, 
In the FederatedDataLoader function, `**kwargs` is passed as a parameter. 
> kwargs = {'num_workers': 1, 'pin_memory': True}

However, the DataLoader function implementation in Pysyft doesn't take `num_workers` and `pin_memory` as arguments as in pytorch. 

> **Hence running this below snippet results in an error:**

> `federated_train_loader = sy.FederatedDataLoader( 
    datasets.MNIST('../data', train=True, download=True,
                   transform=transforms.Compose([
                       transforms.ToTensor(),
                       transforms.Normalize((0.1307,), (0.3081,))
                   ]))
    .federate((bob, alice)), 
    batch_size=args.batch_size, shuffle=True, **kwargs)`

Shouldn't this argument be removed from this function?

**Comment by LaRiffle** [comment]
Hi, this has been fixed in #1982
The change should be already in dev branch, and next week in master
You can now use this code and it should work!

**Comment by sreyafrancis** [comment]
Okay, thanks!
